plugins/modules/nfs.py

```diff
@@ -588,19 +586,21 @@ def _get_nfs_export_from_id(self, nfs_export_id, access_zone):
 
     def _create_nfs_export_create_params_object(self, path):
         try:
-            nfs_export = self.isi_sdk.NfsExportCreateParams(
-                paths=[path],
-                clients=self.module.params['clients'],
-                read_only_clients=self.module.params['read_only_clients'],
-                read_write_clients=self.module.params['read_write_clients'],
-                root_clients=self.module.params['root_clients'],
-                read_only=self.module.params['read_only'],
-                all_dirs=self.module.params['sub_directories_mountable'],
-                description=self.module.params['description'],
-                security_flavors=get_security_keys(
-                    self.module.params['security_flavors']),
-                zone=self.module.params['access_zone'])
-            return nfs_export
+            if not self.module.check_mode:
```

Comment:
Do we need check_mode condition here? Here this method does prepare the payload.

Reply:
handled this by removing.